README.md

```diff
@@ -55,3 +55,12 @@ Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser
 During VM provisioning, all the dependencies are installed. Provisioning steps are documented in vagrantfile.
 
 Once VM is brought up (using the command $ vagrant up), visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Notes for Module-5
+For Part-1 of exercise, execute following command to create container - 
+    docker run -p 5000:5000 --env-file .env todo-app gunicorn --bind 0.0.0.0:5000 todo_app.app:app
+For Part-2 (multi-stage build file), following are the commands - 
```

Comment:
Nice notes, but we're missing instructions for the `docker build` step before we run the container

Reply:
Modified notes to include additional information.